apps/fis/fis-cos-api/fis-cos-api.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: fis-cos-api
+spec:
+  releaseName: fis-cos-api
+  interval: 1m
+  values:
+    nodejs:
+      image: hmctspublic.azurecr.io/fis/cos-api:prod-181a903-20220331113745 #{"$imagepolicy": "flux-system:fis-cos-api"}
+  chart:
+    spec:
+      chart: ./stable/fis-cos-api
+      sourceRef:
+        kind: GitRepository
+        name: hmcts-charts
+        namespace: flux-system
+      interval: 1m
```

Comment:
This file is only needed as declared in `k8s/namespaces/fis/fis-cos-api/fis-cos-api.yaml`, app deployment is still done with fluxV1 which uses /namespaces